src/main/webapp/script.js

```diff
@@ -46,27 +46,36 @@ function updateFeedLinkForCurrentUser() {
   link.setAttribute('href', 'feed.html?id=' + uid);
 }
 
-function updateMatchesLinkForCurrentUser() {
-  let uid = getCurrentUserId();
-  var link = document.getElementById("matchesLink");
+function updateMatchesLinkForCurrentUser(userid) {
+  const link = document.getElementById("matchesLink");
   link.setAttribute('href', 'matches.html?id=' + uid);
 }
 
 window.addEventListener("load", function(){
   function a(a,b){var c=/^(?:file):/,d=new XMLHttpRequest,e=0;d.onreadystatechange=function(){4==d.readyState&&(e=d.status),c.test(location.href)&&d.responseText&&(e=200),4==d.readyState&&200==e&&(a.outerHTML=d.responseText)};try{d.open("GET",b,!0),d.send()}catch(f){}}var b,c=document.getElementsByTagName("*");for(b in c)c[b].hasAttribute&&c[b].hasAttribute("data-include")&&a(c[b],c[b].getAttribute("data-include"));
 });
 
+function getMutualFriends(pmID) {
+  currentUser = getCurrentUserId();
+  fetch('/mutual-friends?userid1=' + currentUser + '&userid2=' + pmID).then(response => response.json()).then((mutualFriends) => {
+    console.log(mutualFriends);
+    return mutualFriends;
```

Comment:
Hmm I don't think `getMutualFriends` will actually return this value, since the `fetch()` occurs asynchronously -- the correct value will be logged, but the returned value will always be `null` or `undefined`.